Products/ZenModel/RRDTemplate.py

```diff
@@ -464,9 +464,13 @@ def getDataSourceInstance(self, id, dsOption):
     def getThresholdClasses(self):
         from Products.ZenModel.MinMaxThreshold import MinMaxThreshold
         from Products.ZenModel.ValueChangeThreshold import ValueChangeThreshold
-        thresholdClasses = [MinMaxThreshold, ValueChangeThreshold]
+        from Products.ZenModel.TrendlineThreshold import TrendlineThreshold
+        thresholdClasses = [MinMaxThreshold, ValueChangeThreshold, TrendlineThreshold]
         for zp in self.dmd.ZenPackManager.packs():
-            thresholdClasses += zp.getThresholdClasses()
+            try:
+                thresholdClasses += zp.getThresholdClasses()
+            except Exception, e:
```

Comment:
Do not stack trace if there is a zenpack we are unable to pull the thresholds from.